nano/lib/rpcconfig.cpp

```diff
@@ -233,10 +233,21 @@ nano::error read_rpc_config_toml (boost::filesystem::path const & data_path_a, n
 	// Parse and deserialize
 	nano::tomlconfig toml;
 
+	std::stringstream config_overrides_stream;
+	for (auto const & entry : config_overrides)
+	{
+		config_overrides_stream << entry << std::endl;
```

Comment:
Could use `"\n"` instead of flushing each time with `std::endl`

Reply:
What does flushing mean for a stringstream?

Reply:
The same for any output stream https://en.cppreference.com/w/cpp/io/basic_ostream/flush:
```
Writes uncommitted changes to the underlying output sequence.
If rdbuf() is a null pointer, does nothing
Otherwise, behaves as an UnformattedOutputFunction (since C++11). After constructing and checking the sentry object, calls rdbuf()->pubsync(). If the call returns -1, calls setstate(badbit).
```

I did some benchmarking to see if there was any meaningful difference:
http://quick-bench.com/BGpm2DCcIS8xk65lUvmHMzQiGLA

Strangely with libstdc++ the `std::endl` is a bit quicker! but libc++ is a lot slower. I doubt it matters here anyway given there won't be many. My comment about using endl after is irrelevant here sorry.

Reply:
Right, probably fine as this is only on startup. But that benchmark is good to know about, esp in performance sensitive code on libc++ 😅